integrations/aws/main.py

```diff
@@ -0,0 +1,203 @@
+        yield fix_unserializable_date_properties(account)
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.CLOUDRESOURCE)
+async def resync_generic_cloud_resource(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    resource_kinds = get_resource_kinds_from_config(kind)
+
+    if len(resource_kinds) == 0:
+        logger.error(f"Resource kinds not found in port app config for {kind}")
+        return
+
+    for kind in resource_kinds:
+        async for batch in resync_cloudcontrol(kind):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.ACM)
+async def resync_acm(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in get_sessions():
+        async for batch in batch_resources(
+            kind,
+            session,
+            "acm",
+            "list_certificates",
+            "CertificateSummaryList",
+            "NextToken",
+        ):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.LOADBALANCER)
+async def resync_loadbalancer(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in get_sessions():
+        async for batch in batch_resources(
+            kind,
+            session,
+            "elbv2",
+            "describe_load_balancers",
+            "LoadBalancers",
+            "NextMarker",
+        ):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.CLOUDFORMATION)
+async def resync_cloudformation(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in get_sessions():
+        async for batch in batch_resources(
+            kind,
+            session,
+            "cloudformation",
+            "list_stacks",
+            "StackSummaries",
+            "NextToken",
+        ):
+            yield batch
+
+
+@ocean.on_resync(kind=ResourceKindsWithSpecialHandling.EC2)
+async def resync_ec2(kind: str) -> ASYNC_GENERATOR_RESYNC_TYPE:
+    async for session in get_sessions():
+        region = session.region_name
+        account_id = await _session_manager.find_account_id_by_session(session)
+        try:
+            async with session.resource("ec2") as ec2:
+                async with session.client("ec2") as ec2_client:
+                    async for page in ec2.instances.pages():
+                        if not page:
+                            continue
+                        page_instances = []
+                        instance_ids = [instance.id for instance in page]
+                        described_instances = await ec2_client.describe_instances(
+                            InstanceIds=instance_ids,
+                        )
+                        for reservation in described_instances["Reservations"]:
+                            for instance in reservation["Instances"]:
+                                instance.update(
+                                    {
+                                        KIND_PROPERTY: kind,
```

Comment:
kind is not needed unless it is a cloud resource

Reply:
I don't know why would the user want to use this, but shouldn't we keep the convection of adding each kind and not add it for some and for some not?